tests/nested/manual/fde-hooks/task.yaml

```diff
@@ -0,0 +1,43 @@
+summary: Check that the fde-setup hooks work
+
+system: [ubuntu-20.04-64]
+
+environment:
+    NESTED_IMAGE_ID: core20-fde-setup
+    NESTED_ENABLE_TPM: false
+    NESTED_ENABLE_SECURE_BOOT: false
+    NESTED_BUILD_SNAPD_FROM_CURRENT: true
+
+environment:
+    GO111MODULE: off
+
+prepare: |
+  echo "Build a kernel snap with the fde-setup hook"
+  # shellcheck source=tests/lib/prepare.sh
+  . "$TESTSLIB/prepare.sh"
+  # shellcheck source=tests/lib/nested.sh
+  . "$TESTSLIB/nested.sh"
+  mkdir ./extra-snaps
+  # build fde-reveal-key hook into the "extra-initrd" dir so that the
+  # nested_create_core_vm picks this up
+  mkdir -p ./extra-initrd/usr/bin/
+  go build -o ./extra-initrd/usr/bin/fde-reveal-key "$TESTSLIB"/fde-setup-hook/fde-setup.go
+  # create fde-setup hook inside the kernel
+  mkdir -p ./extra-kernel-snap/meta/hooks
+  go build -o ./extra-kernel-snap/meta/hooks/fde-setup "$TESTSLIB"/fde-setup-hook/fde-setup.go
+  nested_create_core_vm
+  # start it
+  export NESTED_ENABLE_OVMF=true
```

Comment:
why not define this in the environment section?

Reply:
Indeed, that is slightly silly. Fixed.